src/bundle/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const TRANSFORM_EXTENSIONS = [".jsx", ".cjsx", ".mjsx", ".tsx", ".ctsx", ".mtsx"];
+export const JS_EXTENSION_RE = /^\.([cm]?[tj]sx?|json)$/;
```

Comment:
used in order to transform only javascript files and json. Omit all other extensions like `.png`, `.woff`, `.css` and etc.